src/react-apps/applications/shared/__tests__/profileMenu.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import ProfileMenuComponent from '../src/navigation/main-header/profileMenu';
+
+describe('ProfileMenu', () => {
+  let profileMenu: any;
+  let rendered: any;
+
+  beforeEach(() => {
+    profileMenu = null;
+    rendered = null;
+  });
+
+  it('snapshot should match snapshot with default values', () => {
+    rendered = shallow(
+      <ProfileMenuComponent />,
+    );
+    profileMenu = rendered.shallow();
```

Comment:
Ref comment above, just remove the declaration of the variables, and the `beforEach` and have

```
let rendered = shallow (<ProfileMenuComponent/>);
let profileMenu = rendered.shallow();
expect(profileMenu).toMatchSnapshot();
```

Any reason for calling shallow twice on the component?

Reply:
Material-UI wraps the component with withStyles() so you have to dive() or shallow() the component once more. MUI has created their own createShallow() function, which I've chosen not to use because we also could have more than one wrapper.

Enzyme is working on a dive(depth) coming in a future release.

Reply:
Changed the layout.